infra.bs

```diff
@@ -331,8 +331,8 @@ from the list that match a given condition, or do nothing if none do.
  "<code>b</code>", "<code>ba</code>" ».
 </div>
 
-<p>A <a>list</a> <dfn export for=list,stack,queue,set>contains</dfn> an item if it appears in the
-list.
+<p>A <a>list</a> <dfn export for=list,stack,queue,set lt="contain">contains</dfn> an item if it 
```

Comment:
Nit: I think this doc generally avoids quotes when not necessary. So, probably prefer `lt=contain`.